### ISSUE TITLE
New message view is bloated

### ISSUE THREAD

**Issue by ghost** [thunderbird/thunderbird-android#6827]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.6

### Where did you get the app from?

Google Play

### Android version

13

### Device model

Samsung S20+

### Steps to reproduce

1. Install 6.6 (auto update just rolled out)
2. Read the changelog and changed the density to compact
3. Restarted app (just in case).

### Expected behavior

K-9's message view should remain compact at least as option to facilitate the primary use of reading your mail.

### Actual behavior

The resigned message view is bloated (oppose of compact; no disrespect intended) leaving just 55% of the screen to display the message body (measured by hand so might be slightly off).  I don't see a way to customize it be improve information density. 
You still want to show the sender and subject, of course, but both of those sections are bloated (white space, large font, large "show remote images" button) so not sure what the right "metric" is.  To me this a major UX regression of the primary use case.

top menu: 204px (7%)
subject: 426px (14%)
from/reply/show remote images: 542px (18%)
body: 1722px (55%)
bottom menu: 190px (6%)

![Screenshot_20230417_201608_K-9 Mail](https://user-images.githubusercontent.com/430025/232640304-17fd4705-7e33-461c-a475-41a8dcc33bf2.jpg)

**Comment by allanwind** [comment]
> Making the user interface as compact as possible is not a design goal of K-9 Mail. If this is a requirement for you, maybe you're better off using a different client.

Yes, reading email is the primary use case for an email client.  I am saddened that you care that little about retaining long time users.

**Comment by htsmi** [comment]
I hope that the Mozilla involvement is not turning K-9 into another one of 'those' apps with tiny little islands of text in an ocean of whitespace and zero division between elements. This does not look promising for the K-9 I have used and loved... I know I have been behind on updates, but I was caught by surprise by these changes in a what I thought was a minor upgrade to 6.602, and even changing the 'density' settings to compact did not bring me back to what I would consider an efficient or intuitive use of space. I have already set my system-wide text and element size to small to take advantage of the large high DPI screen I paid for, yet it seems like I have to fight every app on this.

The "show remote images" takes up an absurd amount of space... why? The colored account indicator is gone, the header takes up the same amount of space but shows a fraction of the information. I do not enjoy digging through menus just to see the message time or who the message is addressed and CC'd to. Ironically the message list is now too smooshed together, not enough padding or visual contrast between elements. The subject and sender are now the same text color as the summary, the thread message count looks like it is part of the subject line, the list items are not even consistent heights. The to vs CC indicator for threads is gone. Previously I could distinguish between the elements at a simple glance, now it takes a lot more concentration to make any visual sense of it.

I don't expect the interface to never change, but there needs to be a good rationale if information is going to be more hidden or less visually clear. I am not asking for the interface to be as compact as possible, only around the compromise level we had previously, which was far from 'cluttered'. Email is not instant messaging and should not be reduced to something like it, information like the recipient/CC list is critical for understanding context of messages, and obscuring this increases the chance that people will be hurt by improper responses and sharing of information.

The past few years' interface changes have been excellent, moving folders to the side pane and the quick account switcher have made the app so much more efficient for me to use. I welcome any changes similar to those, but these seem like changes to get on the bandwagon of the 'simplification' that is killing so many efficient and usable programs. 'Cleanness' of the interface cannot be the ultimate goal, sometimes information is ugly but essential! If I wanted the slickest interface I would use the Google mail app, but I use this app every day for work. Any 'slickness' wears off, but the extra time and taps required to do simple tasks and discern basic information eat into my and others' already limited time and energy. 

I think it is critical to clarify where K-9 is headed as a program. Many long time competent to technical users will be lost if this track continues. If the goal as a Mozilla project is to follow in the footsteps of Firefox Android, I would like to know now, as that has lost so much of its openness, ease of use, functionality, and customizability. The hostility towards those seeking privacy has reached such a level that I no longer use the browser, which I once was an evangelist for. Also are things like surreptitious telemetry coming? Will certain features only be possible when logged into a cloud account, and will there be non-free components? There is quite a gap between Mozilla's approach in recent years and those of small open projects like K-9, so I think it is worthwhile to ask some of these questions.
